### DISCUSSION TITLE
Post variables are not transferred

### DISCUSSION THREAD

**Discussion by nobbikolb** [yandeu/five-server#72] [Help / Questions]
Hi,
 I have a question rearding $_POST Variables.
If I start a php-page from Visual Studio Code with five-server no POST-Variables are transfered .
Isset($_POST['submit']) is always false:

<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <link rel="stylesheet" href="./css/form.css">
</head>
<body>
<?php
 if (isset($_POST['submit'])){
    echo "Speichern";
    $Strecke=$_POST['Strecke'];
    echo $Strecke;
 }
?>
<form action="Erfassen.php" method="post">
        <p>
         <label for="Datum">Datum</label>
         <input type="text" name="Datum" id="idDatum" placeholder="Datum"
          autocomplete="on" required>
        </p>
        <p>
            <label for="Strecke">Strecke</label>
            <input type="text" name="Strecke" id="idStrecke">
        </p>

        <p>
            <label for="Kalorien">Kalorien</label>
            <input type="text" name="Kalorien" id="idKalorien">
        </p>

        <label for="Sportart">Sportart</label>
        <select name="Sportart">
            <option value="Radfahren">Radfahren</option>
            <option value="Laufband">Laufband</option>
        </select>
        <input type="submit" name="submit" value="Speichern">
        <input type="reset" name="reset" value="Zurücksetzen">

    </form>  
</body>
</html>

If I start the page directly with the apache-Server it works, so there is no error in the code
Is there any setting in Five-Server to make it work?

Thank you

nobbi

**Comment by yandeu** [comment]
See https://github.com/yandeu/five-server-vscode/issues/26
